helm/defectdojo/values.yaml

```
@@ -131,6 +131,9 @@ monitoring:
 
 annotations: {}
 
+# Add annotations for secret resources
+secretAnnotations: {}
```

I would change it to the following, to have it more future-proof (in case we will need to customize secrets more):
```suggestion
secrets:
  # Add annotations for secret resources
  annotations: {}
```